demos/dashy/.vscode/launch.json

```diff
@@ -8,6 +8,19 @@
             "name": "Dashy-dev",
             "type": "chrome",
             "request": "launch",
+            "url": "file:///${workspaceRoot}/index.html",
+            "runtimeArgs": [
+                "--disable-web-security"
+            ],
+            "webRoot": "${workspaceRoot}",
+            "sourceMapPathOverrides": {
+                "webpack:///C:/Users/{{username}}/git/hpcc-js/": "C:/Users/{{username}}/git/hpcc-js/"
```

Comment:
Why not use `${workspaceRoot}` here as well?

Reply:
Not sure - that code has just been cut and paste around the place (not sure its even needed here...)

Reply:
Removed undeeded code.